content/projects/jam/index.adoc

```diff
@@ -5,6 +5,9 @@ section: projects
 tags:
 - jam
 - jams
+links:
+  googlegroup: 'jenkinsci-jam'
+  gitter: 'jenkinsci/advocacy-and-outreach-sig'
```

Comment:
@alyssat @bitwiseman Is it fine with you? Or should I refer `#jenkins-community` IRC instead?